autoPyTorch/pipeline/tabular_regression.py

```diff
@@ -195,7 +195,7 @@ def _get_pipeline_steps(self, dataset_properties: Optional[Dict[str, Any]],
             ("preprocessing", EarlyPreprocessing()),
             ("network_backbone", NetworkBackboneChoice(default_dataset_properties)),
             ("network_head", NetworkHeadChoice(default_dataset_properties)),
-            ("network", NetworkComponent(default_dataset_properties)),
+            ("network", NetworkComponent()),
```

Comment:
Do you think we can keep the same signature as other components?

Reply:
see reply above to similar issue in tabular classification pipeline